service/todo.py

```
@@ -0,0 +1,49 @@
+    형식: ["todo1", "todo2", "todo3", ...]
+
+    분석 데이터:
+    {analysis}
+    """
+
+    response = client.models.generate_content(
+        model="gemini-1.5-flash",
+        contents=prompt
+    )
+
+    todos = eval(response.text)
```

Ruff (0.14.4)

Use of possibly insecure function; consider using `ast.literal_eval`

(S307)